agent/exec/container/executor.go

```diff
@@ -41,12 +41,35 @@ func (e *executor) Describe(ctx context.Context) (*api.NodeDescription, error) {
 		}
 	}
 
+	// add v1 plugins to 'plugins'
 	addPlugins("Volume", info.Plugins.Volume)
```

Comment:
The reason `docker info` historically had a `Plugins` section is because, there was no other way to query for that information through docker. Now that plugins are a first-class docker citizen, we should not continue down a legacy path of exposing info.

We should document that `Plugins` in `docker info` lists only pluginsv1 and to query for pluginsv2 use `docker plugin ls`. When pluginsv1 is deprecated, we should consider removing `Plugins` from `docker info`.